sdk/main/src/Executable.cc

```diff
@@ -472,6 +471,7 @@ void Executable<SdkRequestType, ProtoRequestType, ProtoResponseType, SdkResponse
   mCurrentGrpcDeadline = mGrpcDeadline.has_value()              ? mGrpcDeadline.value()
                          : client.getGrpcDeadline().has_value() ? client.getGrpcDeadline().value()
                                                                 : DEFAULT_GRPC_DEADLINE;
+  mMirrorNodeIds = getClientMirrorNodeIds(client);
```

Comment:
Couldn't this just be `mMirrorNodeIds = client.getMirrorNetwork();`? Does `getClientMirrorNodeIds()` need to exist as a whole separate function?